src/Components/Categories/CategoryItem.jsx

```diff
@@ -0,0 +1,48 @@
+import axios from "axios";
+import { useEffect } from "react";
+import "./CategoryItem.css";
+import { useData } from "../../Helper/";
+
+const CategoryItem = () => {
+  const {
+    state: { categories },
+    dispatch,
+  } = useData();
+
+  const fetchCategories = async () => {
+    try {
+      const categoriesData = await axios.get("/api/categories");
+      dispatch({ type: "CATEGORIES", payload: categoriesData.data.categories });
+      console.log(categoriesData.data.categories);
+    } catch (error) {
+      console.error(error);
+    } finally {
+      console.log(categories);
+    }
+  };
+
+  useEffect(() => {
+    if (categories.length === 0) fetchCategories();
+  }, []);
+
+  console.log(categories);
+
+  return (
+    <div className="category_items flex">
+      {categories.map((categoryItem) => (
+        <div className="category_product" key={categoryItem.id}>
+          <img
+            src={categoryItem.image}
+            alt="Earphones"
```

Comment:
`alt` should be dynamic for different categories. Currently it's fixed to `Earphones` only

Reply:
done